.azure-pipelines.yaml

```diff
@@ -1,16 +1,13 @@
-trigger:
-  - master
-
 jobs:
-  - job: 'Install_Packages_and_Lint'
+  - job: "Install_Packages_and_Lint"
     pool:
       vmImage: "ubuntu-20.04"
 
     steps:
       - task: UsePythonVersion@0
         displayName: Use python 3.8.3
         inputs:
-          versionSpec: '3.8.3'
+          versionSpec: "3.8.3"
```

Comment:
```suggestion
          versionSpec: "3.8"
```

Reply:
Installing the latest python version won't break anything
The reason why we do it with pipenv is because setting the minimum version to `3.8.3` will not let you create the venv.